README.md

```diff
@@ -47,7 +47,7 @@ impl HelloService for HelloServer {
 
 fn main() {
     let server_handle = HelloServer.spawn("0.0.0.0:0").unwrap();
-    let client = hello_service::Client::new(server_handle.local_addr()).unwrap();
+    let client = hello_service::Client::new(server_handle.dialer()).unwrap();
```

Comment:
Alternatively, we could have the server handle have a method that returns a client.

`server_handle.new_client().unwrap();`

Thoughts?